frontend/src/__tests__/cypress/cypress/support/commands/odh.ts

```diff
@@ -369,6 +523,9 @@ Cypress.Commands.add(
         pathname = pathname.replace(new RegExp(`:${part}\\b`), replacement);
       });
     }
-    return cy.intercept({ method, pathname, query: options?.query }, response);
+    return cy.intercept(
+      { method, pathname, query: options?.query, ...(options?.times && { times: options.times }) },
```

Comment:
Can this be simplified to:
```suggestion
      { method, pathname, query: options?.query, times: options?.times },
```

Reply:
@christianvogt I have tried this initially , but I am getting this error

![Screenshot from 2024-05-10 21-54-05](https://github.com/opendatahub-io/odh-dashboard/assets/99238909/3a7d4214-a47d-4ec4-b4a2-d8ae967c06c2)

Reply:
That's odd of cypress. Thanks!